.github/workflows/npm-publish.yml

```
@@ -0,0 +1,37 @@
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: 'Node.js Package'
```

```suggestion
name: 'Publish npm package'
```

Maybe something like this is clearer?

---

Indeed, actually I just took the example from GitHub Actions docs. :smile: